include/rocksdb/db.h

```diff
@@ -403,6 +403,14 @@ class DB {
     return Get(options, DefaultColumnFamily(), key, value);
   }
 
+  // Returns all the merge operands corresponding to the key. If the
```

Comment:
Need to explain that `slice` points to an array of at least `num_records` and the caller is responsible to allocate and own.